From 0916e2bf15a1aa50c40570f922441ddc73539c1a Mon Sep 17 00:00:00 2001
From: "Eduardo Lima (Etrunko)" <etrunko@redhat.com>
Date: Fri, 28 Apr 2017 17:16:31 -0300
Subject: [PATCH] utils: Remove unused function
 ovirt_rest_xml_node_get_content()

Most uses were removed in dbf8dd85 "Add generic resource parser" and the
last use in 18d7c005 "storage-domain: Parse storage domain status"

Signed-off-by: Eduardo Lima (Etrunko) <etrunko@redhat.com>
---
 govirt/ovirt-utils.c | 19 -------------------
 govirt/ovirt-utils.h |  1 -
 2 files changed, 20 deletions(-)

diff --git a/govirt/ovirt-utils.c b/govirt/ovirt-utils.c
index 44ea7da..951c693 100644
--- a/govirt/ovirt-utils.c
+++ b/govirt/ovirt-utils.c
@@ -91,25 +91,6 @@ ovirt_rest_xml_node_get_content_from_path(RestXmlNode *node, const char *path)
     return content;
 }
 
-G_GNUC_INTERNAL const char *
-ovirt_rest_xml_node_get_content(RestXmlNode *node, ...)
-{
-    va_list args;
-    const char *content;
-
-    g_return_val_if_fail(node != NULL, NULL);
-
-    va_start(args, node);
-
-    content = ovirt_rest_xml_node_get_content_va(node, &args, NULL);
-
-    va_end(args);
-
-    g_warn_if_fail(node != NULL);
-
-    return content;
-}
-
 static gboolean
 _set_property_value_from_type(GValue *value,
                               GType type,
diff --git a/govirt/ovirt-utils.h b/govirt/ovirt-utils.h
index 4fd4164..e786311 100644
--- a/govirt/ovirt-utils.h
+++ b/govirt/ovirt-utils.h
@@ -36,7 +36,6 @@ struct _OvirtXmlElement
 };
 
 RestXmlNode *ovirt_rest_xml_node_from_call(RestProxyCall *call);
-const char *ovirt_rest_xml_node_get_content(RestXmlNode *node, ...);
 gboolean ovirt_rest_xml_node_parse(RestXmlNode *node,
                                    GObject *object,
                                    OvirtXmlElement *elements);
-- 
2.13.6

